src/dev/demo/index.html

```diff
@@ -14,7 +14,7 @@
     </div>
     <div class="title">
       <a
-        href="https://github.com/github/webauthn-json/blob/gh-pages/src/index.ts"
+        href="https://github.com/github/webauthn-json/blob/gh-pages/demo/index.ts"
```

Comment:
Thanks! This should actually be:
```suggestion
        href="https://github.com/github/webauthn-json/blob/main/src/dev/demo/index.html"
```